pkgs/tools/networking/openssh/default.nix

```diff
@@ -11,30 +11,35 @@ in
 {
   openssh = common rec {
     pname = "openssh";
-    version = "9.9p2";
+    version = "10.0p1";
```

Comment:
This should probably be 10.0p2 now with the download URL adjusted.

Reply:
The download URL is actually 10.0p1 still (hence the discussion above on what to call it...)

Reply:
Yeah, that's what I meant ;) The download URL needs to use 10.0p1 instead of `${version}`